lib/srv/termmanager.go

```diff
@@ -53,7 +53,7 @@ type TermManager struct {
 	// remaining is a partially read chunk of stdin data
 	// we only support one concurrent reader so this isn't mutex protected
 	remaining         []byte
-	readStateUpdate   *sync.Cond
+	readStateUpdate   chan bool
```

Comment:
Could we get a brief comment here saying what the channel is for / how it is used?